### PR TITLE
Upgrade to node 20

### PR DESCRIPTION
**Pull request by VincentSmedinga**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,18 @@
 {
-  "name": "design-system-prototypes",
-  "private": true,
   "version": "0.0.0",
+  "author": "City of Amsterdam",
+  "description": "Prototyping environment for the Amsterdam Design System",
+  "license": "EUPL-1.2",
```
Comment (alimpens) [inline package.json:5]: If we specify this license, we probably also need to add the license file to the repo

File: package.json [hunk 2]
```diff
@@ -1,7 +1,18 @@
...
+  "name": "@amsterdam/design-system-prototypes",
```
Comment (alimpens) [inline package.json:6]: We're not going to publish this as a package or anything, right? Doesn't need to be in the @amsterdam namespace I think

File: package.json [hunk 3]
```diff
@@ -1,7 +1,18 @@
...
+  "keywords": [
+    "nl-design-system"
+  ],
```
Comment (alimpens) [inline package.json:9]: Do we want to keep this? This repo is not really part of the NLDS ecosystem imo

File: package.json (hunk 1 above)
Comment (VincentSmedinga) [reply package.json]: Yea, removed this line again.

File: package.json (hunk 2 above)
Comment (VincentSmedinga) [reply package.json]: I hesitated already, it’s gone now.

File: package.json (hunk 3 above)
Comment (VincentSmedinga) [reply package.json]: I thought it could be a showcase for it, but removed it as well.

**Review by alimpens — APPROVED** [review]

**Pull request merged**